include/dlpack/dlpack.h

```diff
@@ -158,6 +158,15 @@ typedef struct {
  * \brief Plain C Tensor object, does not manage memory.
  */
 typedef struct {
+  /*!
+  * \brief The DLPack and ABI versions of the tensor. The exporting library
+  *  should set the DLPack version to DLPACK_VERSION and ABI version to
+  *  DLPACK_ABI_VERSION.
+  */
+  struct {
+    uint8_t dlpack;
+    uint8_t abi;
```

Comment:
Let us define DLPackVersion struct, for backward compatibility, consider put it in the end(after byte_offset), we need to consider the alignment properties of these fields.

Reply:
This will also make followup compiler support easier

Reply:
Done, thanks!